plugins/modules/foreman_os_default_template.py

```diff
@@ -80,42 +80,19 @@
 
 RETURN = ''' # '''
 
-try:
-    from ansible.module_utils.ansible_nailgun_cement import (
-        find_entities_by_name,
-        find_os_default_template,
-        find_operating_system_by_title,
-        naildown_entity_state,
-    )
-
-    from nailgun.entities import (
-        OperatingSystem,
-        OSDefaultTemplate,
-        ProvisioningTemplate,
-        TemplateKind,
-    )
-except ImportError:
-    pass
 
-from ansible.module_utils.foreman_helper import ForemanEntityAnsibleModule
+from ansible.module_utils.foreman_helper import ForemanEntityApypieAnsibleModule
 
 
-def sanitize_os_default_template_dict(entity_dict):
-    # This is the only true source for names (and conversions thereof)
-    name_map = {
-        'operatingsystem': 'operatingsystem',
-        'template_kind': 'template_kind',
-        'provisioning_template': 'provisioning_template',
-    }
-    result = {}
-    for key, value in name_map.items():
-        if key in entity_dict:
-            result[value] = entity_dict[key]
-    return result
+name_map = {
+    'operatingsystem': 'operatingsystem',
+    'template_kind': 'template_kind',
```

Comment:
I "like" that we use `template_kind` here, but `kind` in the provisioning template. Time to consolidate? :)